src/tools/perf/api/libperf.h

```diff
@@ -98,6 +98,61 @@ enum {
 };
 
 
+/*
+ *   +------+                                                        +------+
+ *   |HOST 0|----------------------PEER_TX(AM)---------------------->|HOST 1|
+ *   +------+        +---------+                  +---------+        +------+
+ *      |            |DPU DMN 0|                  |DPU DMN 1|            |
+ *      |            +---------+                  +---------+            |
+ *      |                 |                            |                 |
+ *      |------INIT------>|                            |<------INIT------|
+ *      |                 |----------PEER_INIT-------->|                 |
+ *      |                 |                            |                 |
+ *      |----SEND_REQ---->|                            |<--RECV_REQ(TM)--|
+ *      |                 |---------PEER_TX(AM)------->|                 |
+ *      |                 |----------(TM msg)--------->|                 |
+ *      |<---SEND_CMPL----|                            |---RECV_CMPL---->|
+ *      |                 |                            |                 |
+ *      |-------FIN------>|                            |<------FIN-------|
+ *
+ * (AM) used only when AM communication is configured
+ * (TM) used only when tag matching communication is configured
+ */
+#define UCP_FOREACH_PERF_DAEMON_AM_ID(_macro) \
+    /* 1) No DPU offloading case: this message is used to transfer data \
+     *    between hosts \
+     * 2) DPU offloading case: Sender DPU->Receiver DPU, (optional) AM message \
+     *    to transfer data between DPU peers when AM communication is used. \
+     *    For tag matching this message isn't used */ \
+    _macro(UCP_PERF_AM_ID_PEER_TX,              0) \
```

Comment:
maybe better to define some general `UCP_PERF_AM_ID` in this file, and then start this enum from that value + 1?